diff --git a/node_modules/@semantic-release/github/index.js b/node_modules/@semantic-release/github/index.js
index fa549f0..7cf7f61 100644
--- a/node_modules/@semantic-release/github/index.js
+++ b/node_modules/@semantic-release/github/index.js
@@ -3,7 +3,6 @@
 const {defaultTo, castArray} = require('lodash');
 const verifyGitHub = require('./lib/verify');
 const addChannelGitHub = require('./lib/add-channel');
-const publishGitHub = require('./lib/publish');
 const successGitHub = require('./lib/success');
 const failGitHub = require('./lib/fail');
 
@@ -28,15 +27,6 @@ async function verifyConditions(pluginConfig, context) {
   verified = true;
 }
 
-async function publish(pluginConfig, context) {
-  if (!verified) {
-    await verifyGitHub(pluginConfig, context);
-    verified = true;
-  }
-
-  return publishGitHub(pluginConfig, context);
-}
-
 async function addChannel(pluginConfig, context) {
   if (!verified) {
     await verifyGitHub(pluginConfig, context);
@@ -64,4 +54,4 @@ async function fail(pluginConfig, context) {
   await failGitHub(pluginConfig, context);
 }
 
-module.exports = {verifyConditions, addChannel, publish, success, fail};
+module.exports = {verifyConditions, addChannel, success, fail};
diff --git a/node_modules/@semantic-release/github/lib/success.js b/node_modules/@semantic-release/github/lib/success.js
index e497cc8..5964b4a 100644
--- a/node_modules/@semantic-release/github/lib/success.js
+++ b/node_modules/@semantic-release/github/lib/success.js
@@ -14,7 +14,7 @@ const getReleaseLinks = require('./get-release-links');
 
 module.exports = async (pluginConfig, context) => {
   const {
-    options: {repositoryUrl},
+    options: {repositoryUrl, tagFormat},
     commits,
     nextRelease,
     releases,
@@ -44,8 +44,9 @@ module.exports = async (pluginConfig, context) => {
   } else {
     const parser = issueParser('github', githubUrl ? {hosts: [githubUrl]} : {});
     const releaseInfos = releases.filter((release) => Boolean(release.name));
-    const shas = commits.map(({hash}) => hash);
-
+    const pkgName = tagFormat.replace('@stoplight/spectral-', '').replace('-${version}', '');
+    const pkgNameRegexp =  RegExp(`^[a-z]+\\(${pkgName}\\):`);
+    const shas = commits.filter(({ subject }) => pkgNameRegexp.test(subject)).map(({hash}) => hash);
     const searchQueries = getSearchQueries(`repo:${owner}/${repo}+type:pr+is:merged`, shas).map(
       async (q) => (await octokit.request('GET /search/issues', {q})).data.items
     );
